test-network-function/networking/suite.go

```diff
@@ -39,9 +44,34 @@ import (
 )
 
 const (
-	defaultNumPings = 5
+	CommandPortDeclared = "oc get pod %s -n %s -o json  | jq -r '.spec.containers[%d].ports'"
+	CommandPortListen   = "ss -tulwnH"
+	defaultNumPings     = 5
+	ocCommandTimeOut    = time.Second * 10
+	indexProtocolName   = 0
+	indexPort           = 4
+)
+
+var (
+	expectersVerboseModeEnabled = false
+	PortDeclaredAndListen       []PortDeclared
+	nodeListen                  PortListen
+	protocolName                = ""
+	port                        = ""
+	name                        = ""
+	flag                        = 0
```

Comment:
I'm not sure I like all of these global variables.  We should try to minimize their usage throughout our code in general.  Could we make the functions that find these variables just return them?

Reply:
I agree, fixed.